### ISSUE TITLE
Question about exclusion of official validation sets in Lingshu training data

### ISSUE THREAD

**Issue by jjj929** [alibaba-damo-academy/MedEvalKit#1]
Hi, thanks for open-sourcing Lingshu, it's a great contribution to the community.

I noticed that the training data excludes test sets from VQA-RAD, SLAKE, PathVQA, MIMIC-CXR, etc. I would like to ask whether the official validation sets of SLAKE, PathVQA, IU-Xray and MIMIC-CXR were also excluded during training.

Thanks again!

**Comment by jjj929** [comment]
Furthermore, could you please provide the test set of IU-Xray, as its official split does not seem to exist. Thanks!

**Comment by wwxu21** [comment]
> Furthermore, could you please provide the test set of IU-Xray, as its official split does not seem to exist. Thanks!

Hello, we follow [ReXrank](https://github.com/rajpurkarlab/ReXrank/) for the validation and test splits

**Comment by wwxu21** [comment]
> Thanks for your reply! I also want to ask, since CheXpert-plus does not have the official test set either. Did you also follow [ReXrank](https://github.com/rajpurkarlab/ReXrank/) and use CheXpert-plus official validation set as your test set?
> 
> Thank you once again for your assistance!

Yes, we use CheXpert-plus official validation set as test set

**Comment by jjj929** [comment]
Thank you for your previous clarification!

I also noticed a discrepancy in the dataset sizes reported in Table 25 of the MedEvalKit paper compared to the actual test sets provided in ReXRank. The differences are summarized below:

| Dataset                   | Reports | Questions | Images |
|---------------------------|---------|-----------|--------|
| ReXRank_IUXray_test       | 590     | 590       | 1180   |
| IUXray_test (Table 25)    | **296**       | **296**       | **607**    |
| ReXRank_CheXpertPlus      | 200     | 200       | 234    |
| CheXpertPlus (Table 25)   | **234**     | **234**       | 234    |

**Since Lingshu has already been trained on the training sets of these datasets, I want to ensure that my evaluation of any further fine-tuning is performed on truly unseen data. Therefore, it's important for me to understand how the test sets were defined in MedEvalKit (since MedEvalKit is truly unseen data for Lingshu).**

Could you please confirm whether Table 25 needs any revision, or if the test splits used differ slightly from those provided in the JSON files ([ReXRank_IUXray_test.json](https://github.com/rajpurkarlab/ReXrank/blob/main/data/iu_xray/ReXRank_IUXray_test.json)   and [ReXRank_CheXpertPlus.json](https://github.com/rajpurkarlab/ReXrank/blob/main/data/chexpert_plus/ReXRank_CheXpertPlus.json))?

Thanks again for your help!

**Comment by kenchan0226** [comment]
Thank you for your question. 

For MIMIC-CXR, we download the official test set from the official website: https://physionet.org/content/mimic-cxr/2.1.0/, and we found that the official test set contains 3269 samples, therefore, we follow the official test set instead of the split provided by RexRank.

For IUXray, we download the official test data from the the National Library of Medicine: https://openi.nlm.nih.gov/faq?download=true, we exclude all the test samples that do not provide the source .png images, resulted in 234 test samples. 

Therefore, the numbers of test samples has subtle differences with RexRank. Sorry for the confusion, we will add a remark to the readme of our repo. 



> Thank you for your previous clarification!
> 
> I also noticed a discrepancy in the dataset sizes reported in Table 25 of the MedEvalKit paper compared to the actual test sets provided in ReXRank. The differences are summarized below:
> 
> Dataset	Reports	Questions	Images
> ReXRank_IUXray_test	590	590	1180
> IUXray_test (Table 25)	**296**	**296**	**607**
> ReXRank_CheXpertPlus	200	200	234
> CheXpertPlus (Table 25)	**234**	**234**	234
> ReXRank_MIMICCXR_test	2347	2347	3858
> MIMIC-CXR (Table 25)	**2195**	**2195**	**3681**
> **Since Lingshu has already been trained on the training sets of these datasets, I want to ensure that my evaluation of any further fine-tuning is performed on truly unseen data. Therefore, it's important for me to understand how the test sets were defined in MedEvalKit (since MedEvalKit is truly unseen data for Lingshu).**
> 
> Could you please confirm whether Table 25 needs any revision, or if the test splits used differ slightly from those provided in the JSON files ([ReXRank_IUXray_test.json](https://github.com/rajpurkarlab/ReXrank/blob/main/data/iu_xray/ReXRank_IUXray_test.json), [ReXRank_CheXpertPlus.json](https://github.com/rajpurkarlab/ReXrank/blob/main/data/chexpert_plus/ReXRank_CheXpertPlus.json) and [ReXRank_MIMICCXR_test.json](https://github.com/rajpurkarlab/ReXrank/blob/main/data/mimic-cxr/ReXRank_MIMICCXR_test.json))?
> 
> Thanks again for your help!

**Comment by jjj929** [comment]
Thank you for your clarification. So I want to confirm that the test split of IU-Xray follows ReXRank (since there is no official split), but excludes those test samples for which the source PNG images are not provided on the OpenI official website? And of course, these excluded samples are also not included in Lingshu's training set?

**Comment by kenchan0226** [comment]
Yes, test split of IU-Xray follows ReXRank, but excludes those test samples for which the source PNG images are not provided on the [OpenI official website](https://openi.nlm.nih.gov/faq?download=true). These excluded samples are also not included in Lingshu's training set. Thanks.

> Thank you for your clarification. So I want to confirm that the test split of IU-Xray follows ReXRank (since there is no official split), but excludes those test samples for which the source PNG images are not provided on the [OpenI official website](https://openi.nlm.nih.gov/faq?download=true)? And of course, these excluded samples are also not included in Lingshu's training set?

**Comment by jjj929** [comment]
Thank you very much for the clarification!

I noticed that there is a slight difference in the data size between CheXpertPlus (Table 25) and ReXRank_CheXpertPlus. I believe this could be addressed with just a minor revision to the original paper's Table 25.

That said, I still hope to see a clarification on whether the official validation sets of SLAKE, PathVQA, IU-Xray, and MIMIC-CXR were also excluded during Lingshu training. I believe such information would greatly benefit a range of fine-tuning efforts based on Lingshu! Thanks.
